udata_gouvfr/models.py

```diff
@@ -30,11 +30,13 @@
 OPENFIELD16 = 'openfield16'
 SPD = 'spd'
 TRANSPORT = 'transport'
+BAL = 'bal'
```

Comment:
Keep the blank line after

Reply:
Which one?
There is actually no blank line here! ;)

Reply:
Oops sorry 🙈 Maybe there should have been one ;-) Forget it.